src/Symfony/Component/VarDumper/Caster/MemcachedCaster.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class MemcachedCaster
+{
+    public static function castMemcached(\Memcached $c, array $a, Stub $stub, $isNested)
+    {
+        foreach ($c->getServerList() as $key => $serverData) {
+            $a += array(
+                Caster::PREFIX_VIRTUAL.'servers' => new EnumStub(
```

Comment:
wouldn't this keep only the last server ?

Reply:
please add a test with multiple servers

Reply:
True, will fix it and will add a test.